django/thunderstore/account/models.py

```diff
@@ -0,0 +1,25 @@
+import ulid2
+from django.conf import settings
+from django.db import models
+
+
+class ServiceAccount(models.Model):
+    uuid = models.UUIDField(default=ulid2.generate_ulid_as_uuid, primary_key=True)
+    user = models.OneToOneField(
+        settings.AUTH_USER_MODEL,
+        related_name="service_account",
+        on_delete=models.CASCADE,
+    )
+    owner = models.ForeignKey(
+        "repository.UploaderIdentity",
+        related_name="service_accounts",
+        on_delete=models.CASCADE,
+    )
+
+    @property
+    def nickname(self) -> str:
+        return self.user.first_name
+
+    def delete(self, *args, **kwargs):
```

Comment:
could wrap this in `@transaction.atomic`. Wouldn't want deleting crash after the user object has already been deleted